tests/e2e/autoscale/00-assert.yaml

```diff
@@ -6,11 +6,17 @@ status:
   readyReplicas: 1
 
 ---
-apiVersion: autoscaling/v1
+apiVersion: autoscaling/v2beta2
 kind: HorizontalPodAutoscaler
 metadata:
   name: simplest-collector
 spec:
   minReplicas: 1
   maxReplicas: 2
-
+  metrics:
```

Comment:
Should we keep a test for the default behavior?

Reply:
Added a new OpenTelemetryCollector that does not set `targetCPUUtilization` for the default.